tests/NuGet.IndexingTests/VerifiedPackagesParsingTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Indexing;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace NuGet.IndexingTests
+{
+    public class VerifiedPackagesParsingTests
```

Comment:
Testing the `Load` method would require mocks that would complicate the test for little gain. Since the `Load` method is only a convenience method, I think testing the `Parse` method directly is fine. Thoughts?